arch/sim/src/sim/up_hcitty.c

```diff
@@ -58,7 +57,7 @@ union bt_hdr_u
   struct bt_hci_iso_hdr_s iso;
 };
 
-struct bthcitty_s
```

Comment:
Why change it from bthcitty everywhere? It's a Bluetooth HCI device not a generic HCI interface.

Reply:
Because the usage is inconsistent. For example, config use SIM_HCITTY and file name use up_hcitty.c.

Reply:
why not just update those then so it is clear?

Reply:
Ok, done.